libs/cardano-protocol-tpraos/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-protocol-tpraos`
 
+## 1.1.0.0
```

Comment:
If you run `git tag --list`:
```
$ git tag --list | grep praos
cardano-protocol-tpraos-1.0.0.0
```
you will see that the latest version that was released to CHaPs was `1.0.0.0`, which means you are updating the wrong section of the changelog. It should be the section below that needs to be updated.

That being said, if it was really a breaking chnage, then the section below would have to be renamed from `1.0.1.0` -> `1.1.0.0`. But because this change is non-breaking, this is not the case